### PR TITLE
test: add missing packages and install test

### PR DESCRIPTION
**Pull request by callmehiphop**
Relates to https://github.com/googleapis/nodejs-pubsub/issues/526#issuecomment-471020084

- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: system-test/fixtures/sample/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build"
```
Comment (JustinBeckwith) [inline system-test/fixtures/sample/tsconfig.json]: Should we try setting `types: [ "node" ]` here to see what happens?  Maybe to repair https://github.com/googleapis/nodejs-pubsub/issues/526?

Comment (JustinBeckwith) [reply system-test/fixtures/sample/tsconfig.json]: Also, skipLibCheck?  https://github.com/googleapis/nodejs-pubsub/issues/532

Comment (callmehiphop) [reply system-test/fixtures/sample/tsconfig.json]: SGTM! Should we consider testing against yarn pnp too?

Comment (JustinBeckwith) [reply system-test/fixtures/sample/tsconfig.json]: Eh, other priorities right now :) If it's easy, go fer it

**Pull request merged**